ios/SalesforceReact/SFOauthReactBridge.m

```diff
@@ -53,6 +53,17 @@ @implementation SFOauthReactBridge
 RCT_EXPORT_METHOD(logoutCurrentUser:(NSDictionary *)args callback:(RCTResponseSenderBlock)callback)
 {
     [SFSDKReactLogger d:[self class] format:@"logoutCurrentUser: arguments: %@", args];
+
+    __block id observerRef;
+    id observer = [[NSNotificationCenter defaultCenter]
+                   addObserverForName:kSFNotificationUserDidLogout
+                   object:nil
+                   queue:[NSOperationQueue mainQueue] usingBlock:^(NSNotification *note) {
+        callback(@[[NSNull null], @"OK"]);
+        [[NSNotificationCenter defaultCenter] removeObserver:observerRef];
```

Comment:
We need to remove the observer once it fired.

Reply:
Do we need this on `Android` too?

Reply:
Android already does the right thing: https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/dev/libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/bridge/SalesforceOauthReactBridge.java#L83